integration/localnet/bootstrap.go

```diff
@@ -317,64 +317,15 @@ func prepareServiceDirs(role string, nodeId string) (string, string) {
 func prepareService(container testnet.ContainerConfig, i int, n int) Service {
 	dataDir, profilerDir := prepareServiceDirs(container.Role.String(), container.NodeID.String())
 
-	service := Service{
-		Image: fmt.Sprintf("localnet-%s", container.Role),
-		Command: []string{
-			fmt.Sprintf("--nodeid=%s", container.NodeID),
-			"--bootstrapdir=/bootstrap",
-			"--datadir=/data/protocol",
-			"--secretsdir=/data/secret",
-			"--loglevel=DEBUG",
-			fmt.Sprintf("--profiler-enabled=%t", profiler),
-			fmt.Sprintf("--tracer-enabled=%t", tracing),
-			"--profiler-dir=/profiler",
-			"--profiler-interval=2m",
-		},
-		Volumes: []string{
-			fmt.Sprintf("%s:/bootstrap:z", BootstrapDir),
-			fmt.Sprintf("%s:/profiler:z", profilerDir),
-			fmt.Sprintf("%s:/data:z", dataDir),
-		},
-		Environment: []string{
-			// https://github.com/open-telemetry/opentelemetry-specification/blob/v1.12.0/specification/protocol/exporter.md
-			"OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=http://tempo:4317",
-			"OTEL_EXPORTER_OTLP_TRACES_INSECURE=true",
-			"BINSTAT_ENABLE",
-			"BINSTAT_LEN_WHAT",
-			"BINSTAT_DMP_NAME",
-			"BINSTAT_DMP_PATH",
-		},
-		Labels: map[string]string{
-			"com.dapperlabs.role": container.Role.String(),
-			"com.dapperlabs.num":  fmt.Sprintf("%03d", i+1),
-		},
-	}
+	service := defaultService(container.Role.String(), dataDir, profilerDir, i)
+	service.Command = append(service.Command, []string{
```

Comment:
skip creating the slice, you're appending a single item here

Reply:
that one is left for the convenience of the next person who would want to add a new arg here.

Reply:
I still don't see the point of creating the slice first, then "...".  append is a vararg built-in function

https://go.dev/play/p/aoSfOM6bxkU